packages/node_modules/@webex/redux-module-spaces/src/actions.js

```diff
@@ -178,17 +183,22 @@ export function updateSpaceWithActivity(activity, isSelf, isReadable = false) {
 export function fetchSpace(sparkInstance, space) {
   return (dispatch) => {
     dispatch(storeInitialSpace(space.id));
+    dispatch(createConversationBegin());
 
     return sparkInstance.internal.conversation.get(space, {
-      activitiesLimit: 25,
+      activitiesLimit: 40,
       participantsLimit: -1,
       participantAckFilter: 'all',
-      includeParticipants: true
+      includeParticipants: true,
+      latestActivity: true
```

Comment:
`lastActivity` was a key:value from `redux-module-conversation`. Moved here for consistency as well.